figures/views.py

```diff
@@ -272,21 +272,19 @@ class CourseDetailsViewSet(CommonAuthMixin, viewsets.ReadOnlyModelViewSet):
 
     '''
     model = CourseOverview
-    # queryset = CourseOverview.objects.all()
     pagination_class = FiguresLimitOffsetPagination
     serializer_class = CourseDetailsSerializer
     filter_backends = (DjangoFilterBackend, )
     filter_class = CourseOverviewFilter
 
     def get_queryset(self):
-        print('CourseDetailsViewSet.get_queryset called')
         site = django.contrib.sites.shortcuts.get_current_site(self.request)
         queryset = figures.sites.get_courses_for_site(site)
         return queryset
 
     def retrieve(self, request, *args, **kwargs):
-        print('CourseDetailsSerializer.retrieve called')
         # NOTE: Duplicating code in GeneralCourseDataViewSet. Candidate to dry up
+        # Make it a decorator
```

Comment:
Just a note here since I'm in views a lot. retrieving the site from the course id string is a perfect candidate for a decorator